pt/lessons/basics/documentation.md

```diff
@@ -1,5 +1,5 @@
 ---
-version: 1.1.0
+version: 1.1.1
```

Comment:
```suggestion
version: 1.1.0
```

We try to the version in sync with the english version. So we should only bump the when the english was also updated. In this case, as we are only fixing spelling and improve the lastest translation there's no need to update the version.

Reply:
I didn't know that, thanks for letting me know! I'll put it back to `1.1.0` (: